apps/champions/lib/champions/battle/simulator.ex

```diff
@@ -621,10 +621,40 @@ defmodule Champions.Battle.Simulator do
     [caster.id]
   end
 
-  defp choose_targets_by_strategy(caster, %{type: "all", target_allies: target_allies}, state),
+  defp choose_targets_by_strategy(
+         caster,
+         %{type: %{"lowest" => stat}} = targeting_strategy,
+         state
+       ) do
+    filter_and_choose_units_by_stat(
+      state.units,
+      stat,
+      targeting_strategy.count,
+      caster,
+      targeting_strategy.target_allies,
+      :desc
+    )
+  end
+
+  defp choose_targets_by_strategy(
+         caster,
+         %{type: %{"highest" => stat}} = targeting_strategy,
+         state
+       ) do
+    filter_and_choose_units_by_stat(
```

Comment:
```suggestion
  defp choose_targets_by_strategy(caster, %{type: %{"highest" => stat}} = targeting_strategy, state) do
```

Reply:
suggestion got suggested wrongly :P you will need to make this change manually

Reply:
lol I hate when that happens

Reply:
yeah it's also uneditable! sucks